bin/main.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../lib/board.rb'
+require_relative '../lib/player.rb'
+require_relative '../lib/game.rb'
+require_relative '../lib/module.rb'
+
+include BoardLogic
+include UI
+
+instructions
+
+board = Board.new
+show_board(board)
+value = choose_sign
```

Comment:
Instead of calling this `value` and `value2`, why don't we make it more intuitive. These should be the `signs` of `player1` and `player2` right? So maybe we can use `sign1` and `sign2` as variable names.